lib/StatusIndicator/styles.scss

```diff
@@ -2,30 +2,55 @@
    Config
    ========================================================================== */
 
-$status-indicator-size: 1em !default;
 $status-indicator-label-color--text: rgb(111, 121, 129) !default;
 
 /* ==========================================================================
    Styles
    ========================================================================== */
 
 .status-indicator {
-  display: inline-block;
-  vertical-align: middle;
   color: $status-indicator-label-color--text;
+  .due-date {
```

Comment:
Most styles should use the BEM naming format for classes. For example if this is passed as the child component then maybe something like `status-indicator__content`. 

If you feel like the new description suits this better then maybe switch it round and have the description as the child component and the date as a prop on the component.

What do you think?